server/user/user.router.js

```diff
@@ -63,5 +64,15 @@ function getPostsForUser(req, res) {
     );
 }
 
+function update(req, res) {
```

Comment:
It works, but it might be better to find by id and only update necessary fields (username should never be updated)
Might cause security issues
Do not modify, just a future suggestion